plugins/sqlfluff-plugin-example/src/example/plugin_default_config.cfg

```diff
@@ -0,0 +1,2 @@
+[sqlfluff:rules:Example_L001]
```

Comment:
This is a potentially tricky thought, but what if a plugin provides configuration for **core** rules? Which one wins, the core default or the plugin default? What if multiple plugins provide defaults for the same core rule?

If this seems like a legitimate worry, it could be addressed in a later PR; not necessary for the first version.

Reply:
I'm going to look into preventing plugins from providing "core" configuration in their default config file

Reply:
I looked into that, and opted for changing the docs instead. It was adding quite a bit of complexity and I wasn't sure that it wouldn't bring other edge cases.

Reply:
That's fine. It's an unlikely situation anyway. Professional pessimist here. 😁